wireapp/wire-android-reloaded
# feat(calling): Display active speaker indication(AR-1766)

<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-1766" title="AR-1766" target="_blank"><img alt="Subtask" src="https://wearezeta.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/10816?size=medium" />AR-1766</a>  Display active speaker indication
  </td></table>
  <br />
 


























----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Description
This PR aims to show the active speaker in an ongoing call by making his tile border blue (see screenshot)

### Dependencies (Optional)

Blocked By :
[feat(calling): get active speaker in an ongoing call(AR-1766)
](https://github.com/wireapp/kalium/pull/700)

Needs releases with:


### Testing

#### Test Coverage (Optional)


#### How to Test

Make a call and start speaking, you should see your tile border and name changed to blue

### Attachments (Optional)

![Screenshot_20220721-164138](https://user-images.githubusercontent.com/18124919/180266647-e92563a7-7d65-4c19-b0f8-bc81556140cc.png)
![Screenshot_20220721-164208](https://user-images.githubusercontent.com/18124919/180266656-1f08f4ab-f24a-4492-9239-712a8487afce.png)


_Attachments like images, videos, etc. (drag and drop in the text box)_

----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
